### PR TITLE
Get rid of 'six' usage

### PR DESCRIPTION
**Pull request by Polina-Gubina**
Getting rid of 'six' usage which leads to stop compatibility with py2

### CODE REVIEW

**Review by anton-sidelnikov — DISMISSED** [review]

File: otcextensions/common/utils.py [hunk 1]
```diff
@@ -21,12 +21,11 @@
 # import json
 import os
 # import re
-# import six.moves.urllib.parse as urlparse
+# import urllib.parse as urlparse
 # import sys
```
Comment (anton-sidelnikov) [inline otcextensions/common/utils.py]: do we need  to store that commented items?

Comment (gtema) [reply otcextensions/common/utils.py]: nope, we do not need them. I forgot to comment on that also.

Comment (Polina-Gubina) [reply otcextensions/common/utils.py]: fixed

**Comment by Polina-Gubina** [comment]
recheck

**Review by anton-sidelnikov — APPROVED** [review]

**Review by gtema — APPROVED** [review]

**Pull request merged**